ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/clientlibs/editorhook/js/replacehook.js

```diff
@@ -103,7 +104,8 @@
                 }
 
                 if (!(keyword.length > 0) || isKeywordFound) {
-                    if ((!cannotBeReplacedWith.includes(componentType) && editableType != componentType)
+                    if ((c.componentConfig.group === 'Core Components Examples - Adaptive Form')
```

Comment:
@nit23uec , do we check on component group name , is it a practice that we follow, otherwise please suggest

Reply:
-1 for having group name in code as it varies from customer to customer.

Reply:
To detect if its a form component, we can put a check on basis of fieldType is present on not from what we support in schema.

Reply:
<img width="1728" alt="image" src="https://user-images.githubusercontent.com/18582880/232989617-ca172153-f18a-4a66-ba0d-ae534e6d451f.png">
Apart from group name, we have templatePath, resourceType etc but these params can also be updated by the customer. But we don't have any definitive parameter to check if it is adaptive form for not.

Reply:
We can't check group name since group name is customer specific, you need to check for fieldType